libs/metagen/src/mdk_python/types.rs

```diff
@@ -45,7 +44,9 @@ pub fn visit_type(
         TypeNode::Union { .. } | TypeNode::Either { .. } => {
             visit_union_or_either(tera, memo, tpe, tg)?.hint
         }
-        _ => bail!("Unsupported type {:?}", tpe.type_name()),
+        // TODO: base64
+        TypeNode::File { .. } => "str".to_string(),
+        TypeNode::Any { .. } => "Any".to_string(),
```

Comment:
Maybe we can use a simple alias? `Base64String = str`

Reply:
Yeah, this would need specialized handling. Base64 a 25MiB file wouldn't make sense. Maybe exposing the file as a preopen to the modules? Not sure.